<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog_range</class>
 <widget class="QDialog" name="Dialog_range">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_ok">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_cancel">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>240</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xfrom">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_xto">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>60</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yfrom">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>130</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>-10</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_yto">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>81</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xfrom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x from =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_xto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>60</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>x to =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yfrom">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>130</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y from =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_yto">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>130</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>y to =</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>30</y>
     <width>211</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Please enter only numbers below</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
